web/client/components/data/featuregrid/filterRenderers/AttributeFilter.jsx

```diff
@@ -21,7 +22,10 @@ class AttributeFilter extends React.PureComponent {
         value: PropTypes.any,
         column: PropTypes.object,
         placeholderMsgId: PropTypes.string,
-        tooltipMsgId: PropTypes.string
+        tooltipMsgId: PropTypes.string,
+        operator: PropTypes.string,
+        type: PropTypes.string,
+        isWithinAttrTbl: PropTypes.bool
```

Comment:
i think that this should be removed in the future since also attribute table in dashboard should inherit timerange changes